### ISSUE TITLE
;;upvote command

### ISSUE THREAD

**Issue by RealKC** [Frederikam/FredBoat#151]
Suggested by @Truzo#6331 in #suggestions:
>[2:10 PM] Truzo: Is ;upvote already suggested? Would be nice :smiley: like typ in the command, you get the list of all songs and you can do like ;upvote 2. (I mis the good old Soundrop times on Spotify :3 )
[2:11 PM] Banhammer will never cheat on me: what would upvote's function be tho?
[2:14 PM] Truzo: Upvote songs. Like you're reading the queu, you wanna hear a song that's like on position 20 earlier, you do ;upvote 20 and it will go to place 19.
[2:14 PM] Truzo: Now you can only shuffle or play in order of adding. It's really just like soundrop.

He didn't really mention wether the song that used to be in (using his examples) is on position 19 gets deleted or moved to the end of the queue or moved on position 20, I'd say on position 20. Could allow people who have the Manage Messages permission to upvote a song more than a single position.

Usage
`;;upvote n` - moves song on position `n` a position
`;;upvote n -m` - moves song on postion `n` by `m` positions, `m` should have a limit tho.

Not sure `upvote` is the best name for it, we could call it `soundrop` since I guess more people will understand its purpose.

**Comment by Frederikam** [comment]
;;upvote sounds like a very misleading name, nor do I think this would be
used by a lot of people. I don't see myself using it.

**Comment by RealKC** [comment]
I agree `upvote` is a misleading name, I see myself using it, but in rare cases.

**Comment by napstr** [comment]
I recommend touching this after #210 and #245 are resolved, as they change the behaviour/refactor relevant classes for this.
